homeassistant/components/matter/manifest.json

```diff
@@ -6,6 +6,6 @@
   "dependencies": ["websocket_api"],
   "documentation": "https://www.home-assistant.io/integrations/matter",
   "iot_class": "local_push",
-  "requirements": ["python-matter-server==6.3.0"],
+  "requirements": ["python-matter-server==6.5.0b0"],
```

Comment:
We will release 6.5.0 of the library soon. Then we can do a PR to bump the lib version and after that rebase this PR